tests/src/CoreMangLib/system/span/BasicSpanTest.cs

```diff
@@ -17,10 +17,257 @@ static int Sum(Span<int> span)
 
     static void Main()
     {
+        Test(() => CanAccessItemsViaIndexer(), "CanAccessItemsViaIndexer");
+
+        Test(() => CanCopyValueTypesWithoutPointersToSlice(), "CanCopyValueTypesWithoutPointersToSlice");
+        Test(() => CanCopyValueTypesWithPointersToSlice(), "CanCopyValueTypesWithPointersToSlice");
+        Test(() => CanCopyReferenceTypesToSlice(), "CanCopyReferenceTypesToSlice");
+
+        Test(() => CanCopyValueTypesWithoutPointersToArray(), "CanCopyValueTypesWithoutPointersToArray");
+        Test(() => CanCopyValueTypesWithPointersToArray(), "CanCopyValueTypesWithPointersToArray");
+        Test(() => CanCopyReferenceTypesToArray(), "CanCopyReferenceTypesToArray");
+
+        Test(() => CanCopyValueTypesWithoutPointersToUnmanagedMemory(), "CanCopyValueTypesWithoutPointersToUnmanagedMemory");
+        Test(() => MustNotCopyValueTypesWithPointersToUnmanagedMemory(), "MustNotCopyValueTypesWithPointersToUnmanagedMemory");
```

Comment:
I believe you need to return a 100/-1 error code from `Main` to indicate that the test passed/failed.